pkg/kapis/openpitrix/v1/application.go

```diff
@@ -16,112 +16,90 @@
  * /
  */
 
-package openpitrix
+package v1
 
 import (
 	"fmt"
 	"github.com/emicklei/go-restful"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"k8s.io/api/core/v1"
 	"k8s.io/klog"
 	"kubesphere.io/kubesphere/pkg/constants"
 	"kubesphere.io/kubesphere/pkg/models"
 	"kubesphere.io/kubesphere/pkg/models/openpitrix"
-	"kubesphere.io/kubesphere/pkg/models/resources/v1alpha2"
 	"kubesphere.io/kubesphere/pkg/server/errors"
 	"kubesphere.io/kubesphere/pkg/server/params"
-	"kubesphere.io/kubesphere/pkg/simple/client"
 	"net/http"
 )
 
-func ListApplications(req *restful.Request, resp *restful.Response) {
-	limit, offset := params.ParsePaging(req.QueryParameter(params.PagingParam))
-	namespaceName := req.PathParameter("namespace")
-	conditions, err := params.ParseConditions(req.QueryParameter(params.ConditionsParam))
-	orderBy := req.QueryParameter(params.OrderByParam)
-	reverse := params.ParseReverse(req)
-
-	if orderBy == "" {
-		orderBy = "create_time"
-		reverse = true
-	}
+func (h *Handler) ListApplications(request *restful.Request, response *restful.Response) {
```

Comment:
we are moving restful handlers to kapis/openpitrix/v1/handler.go, which is more clear.